dev-ml/stringext/stringext-1.4.0.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+OASIS_BUILD_TESTS=1
+
+inherit oasis
+
+DESCRIPTION="Extra string functions for OCaml"
+HOMEPAGE="https://github.com/rgrinberg/stringext"
+SRC_URI="https://github.com/rgrinberg/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND="
+	test? ( dev-ml/qcheck[ounit(-)] )
+	>=dev-lang/ocaml-4
+"
+RDEPEND=""
```

Comment:
probably; oasis.eclass adds a := dep on ocaml, meaning that if ocaml is downgraded it needs to be rebuilt, so this dep effectively makes it a RDEPEND on ocaml-4 actually, hence better move it to RDEPEND (with a := just for clarity)